src/components/Pagination/Pagination.tsx

```diff
@@ -30,17 +32,25 @@ export const Pagination: React.FC<Props> = ({
           onClick={handleLeftButtonClick}
           disabled={currentPage === 1}
         >
-          {'<'}
+          <Icon icon="chevron-thin-left" className={styles.pagination__icon} />
         </button>
       </li>
 
-      {getPages(TOTAL_PAGES_NUMBER).map((page, index) => (
+      {getPages(totalPagesNumber).map((page, index) => (
         <li className={styles.pagination__item} key={index}>
           <button
             className={classNames(styles.pagination__button, {
               [styles['pagination__button--active']]: page === currentPage,
             })}
-            onClick={() => setCurrentPage(page)}
+            onClick={() => {
```

Comment:
please try no to use arrow functions in the body of the component under return statement as long as it does not have any parameters provided explicitly

try to move handler to separate method and simply pass a reference to it

onClick={onClickHandler} or something like that